modules/standard/CTypes.chpl

```diff
@@ -509,6 +510,10 @@ module CTypes {
     return __primitive("cast", t, x);
   }
   @chpldoc.nodoc
+  inline operator :(x:c_ptrConst(c_uchar), type t:c_ptrConst(c_char)) {
```

Comment:
Isn't this covered by the above cast between c pointers on line 496? Why add this one?

Reply:
removed